[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:1051 Starting new HTTPS connection (1): petstore.swagger.io:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:546 https://petstore.swagger.io:443 "PUT /v2/pet/ HTTP/11" 200 None
[32mINFO    [0m APIPetstoreTests.test_pets:test_pets.py:33 API call done